### ISSUE TITLE
[Proposal] Listing multiple exposure category for one dataset

### ISSUE THREAD

**Issue by matamadio** [GFDRR/rdl-standard#273]
## What is the context or reason for the change?
There are "asset exposure" datasets that are made of several components. We want to list them all together without splitting the dataset.

## Why is this not covered by the existing model?
Currently only one exposure category can be selected for one dataset.

## What is your proposed change?
Make exposure category into an array: a dataset can include multiple categories, each one specified by one or more exposure metrics.

## Can you provide an example?
[Afghanistan-asset-exposure](https://datacatalog.worldbank.org/search/dataset/0050638/Afghanistan-asset-exposure) includes buildings, infrastructures, agriculture, population.
We want to be able to group them into the same "Exposed Asset" dataset/page, e.g. buildings;infrastructures;agriculture;population

**Comment by duncandewhurst** [comment]
@matamadio `exposure.category` is still a string on `0.2-dev` and `Exposure_item.category` is still a string on `0.3-dev` so I'm not clear on what change was implemented for this issue. However, since `exposure` is now an array of `Exposure_item` objects (on `0.3-dev`), I'm assuming that this change is no longer required. Please confirm.

**Comment by matamadio** [comment]
@duncandewhurst correct, as from other comments, instead of having nested propierties within one item, we rather allow multiple exposure items (each with own propierties).

**Comment by matamadio** [comment]
Related to #339

**Comment by duncandewhurst** [comment]
Great. Thanks for clarifying. I'll close this issue, but feel free to reopen if needed.
